bus-mapping/src/state_db.rs

```diff
@@ -14,7 +14,12 @@ lazy_static! {
 
 /// Memory storage for contract code by code hash.
 #[derive(Debug, Clone)]
-pub struct CodeDB(pub HashMap<Hash, Vec<u8>>);
+pub struct CodeDB {
+    /// contract address to code hash
+    pub address_hash: HashMap<Address, Hash>,
```

Comment:
I see that this hash is filled when calling `CodeDB::insert`, but where is it being read?
Can't we get the same information from `StateDB::get_account(address).code_hash`?

Reply:
This is a legacy change of this pr. We will revert this change immediately.

Reply:
oh yeah! thanks for the catch! that is used in previous implementation. I reverted this unnecessary change.